test/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "jsx": "react-jsx",
     "types": ["mocha", "chai", "sinon", "sinon-chai"],
     "rootDir": "..",
+    "esModuleInterop": true
```

Comment:
This isn't needed because the base config already has this set.

Reply:
I was getting an error on VSCode that suggest me to add this option in the config. 
I tried today without this option and everything is working 😑